src/OrbitClient.php

```diff
@@ -38,10 +43,12 @@ class OrbitClient
     public function __construct(
         Client $client,
         CacheItemPoolInterface $cache,
+        string $baseCacheDir,
```

Comment:
@BPScott Because I have added a mandatory parameter, this is a breaking change. Should I bump the version number to 1.4.0?